pkg/agent/proxy/proxier.go

```diff
@@ -978,9 +1013,24 @@ func NewProxier(
 		p.endpointsConfig = config.NewEndpointsConfig(informerFactory.Core().V1().Endpoints(), resyncPeriod)
 		p.endpointsConfig.RegisterEventHandler(p)
 	}
+	if p.multiclusterEnabled {
+		// We register event handler here since serviceCIDRIPv4 is only required
```

Comment:
Probably remove "We register event handler here since", and just say: "serviceCIDRIPv4 is required only when Antrea Multi-cluster is enabled".

Reply:
Done